docs/Using-Fleet/Adding-hosts.md

```diff
@@ -141,15 +142,40 @@ To retrieve the enroll secret, use the "Add New Host" dialog in the Fleet UI or
 
 If your organization has a robust internal public key infrastructure (PKI) and you already deploy TLS client certificates to each host to uniquely identify them, then osquery supports an advanced authentication mechanism that takes advantage of this. Fleet can be fronted with a proxy that will perform the TLS client authentication.
 
-### Deploy the TLS certificate that osquery will use to communicate with Fleet
+### Provide the TLS certificate that osquery will use to communicate with Fleet
 
 When Fleet uses a self-signed certificate, osquery agents will need a copy of that certificate in order to authenticate the Fleet server. If clients connect directly to the Fleet server, you can download the certificate through the Fleet UI. From the main dashboard (`/hosts/manage`), click **Add New Host** and **Fetch Certificate**. If Fleet is running behind a load-balancer that terminates TLS, you will have to talk to your system administrator about where to find this certificate.
 
 It is important that the CN of this certificate matches the hostname or IP that osqueryd clients will use to connect.
 
 Specify the path to this certificate with the `--tls_server_certs` flag when you launch osqueryd.
 
-### Launching osqueryd
+### Configure and launch osquery
+
+In order for osqueryd to connect to the fleet server, there are some flags that need to be set:
+
+```
+ --enroll_secret_path=/etc/osquery/enroll_secret 
+ --tls_server_certs=/etc/osquery/fleet.crt
+ --tls_hostname=fleet.example.com 
+ --host_identifier=uuid 
+ --enroll_tls_endpoint=/api/v1/osquery/enroll 
+ --config_plugin=tls 
+ --config_tls_endpoint=/api/v1/osquery/config 
+ --config_refresh=10 
+ --disable_distributed=false
+ --distributed_plugin=tls 
+ --distributed_interval=10 
+ --distributed_tls_max_attempts=3 
+ --distributed_tls_read_endpoint=/api/v1/osquery/distributed/read 
+ --distributed_tls_write_endpoint=/api/v1/osquery/distributed/write 
+ --logger_plugin=tls 
+ --logger_tls_endpoint=/api/v1/osquery/log 
+ --logger_tls_period=10
+ ```
+These can be specified directly in the command-line, or saved to a flag file. 
+
+#### Launching osqueryd using command-line flags
```

Comment:
```suggestion
#### Launching osquery using command-line flags
```

Reply:
I feel that `osqueryd` is the correct use here since it's specifically the daemon that's being launched.